CONTRIBUTING.md

```diff
@@ -0,0 +1,231 @@
+temporary ban from the repository.
+
+## Pull Requests
+
+Good pull requests &mdash; spelling corrections, revisions, new sections &mdash;
+are a huge help. Pull requests should remain focused and not contain commits not
+related to what you are contributing.
+
+__Ask first__ before embarking on any __significant__ pull request such as add new sections,
+changing the layout, or heavily revising existing sections; otherwise, you risk spending a
+lot of time working on something that Charmed HPC's maintainers may not want to merge into the
+documentation! For trivial changes, or contributions that do not require a large amount of time,
+you can go ahead and make a pull request.
+
+Adhering to the following process is the best way to get your contribution accepted into
+Charmed HPC's documentation:
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) the project, clone your fork,
+   and configure the remotes:
+
+   ```shell
+   # Clone your fork of the repo into the current directory
+   $ git clone git@github.com:<your-username>/docs.git
+
+   # Navigate to the newly cloned directory
+   $ cd docs
+
+   # Assign the original repo to a remote called "upstream"
+   $ git remote add upstream git@github.com:charmed-hpc/docs.git
+   ```
+
+2. If you cloned a the documentation a while ago, pull the latest changes from the
+upstream Charmed HPC documentation repository:
+
+   ```bash
+   $ git checkout main
+   $ git pull upstream main
+   ```
+
+3. Create a new topic branch off the main development branch to
+   contain your feature, change, or fix:
+
+   ```bash
+   $ git checkout -b <topic-branch-name>
+   ```
+
+4. Ensure that your changes pass all tests:
+
+    ```shell
+    # Check links
+    $ make linkcheck
+
+    # Check spelling
+    $ make spelling
+
+    # Check inclusive language
+    $ make woke
+
+    # Check accessibility
+    $ make pa11y
+
+    # Ensure style guide compliance
+    $ make vale
+    ```
+
+5. Commit your changes in logical chunks to your topic branch.
+
+   Our project follows the
+   [Conventional Commits specification, version 1.0.0](https://www.conventionalcommits.org/en/v1.0.0/).
+   You can use Git's
+   [interactive rebase](https://help.github.com/articles/about-git-rebase/) feature to
+   tidy up your commits before pushing them to your origin branch.
+
+6. Locally merge or rebase the upstream development branch into your topic branch:
+
+   ```shell
+   git pull [--rebase] upstream main
+   ```
+
+7. Push your topic branch up to your fork:
+
+   ```shell
+   git push origin <topic-branch-name>
+   ```
+
+8. [Open a Pull Request](https://help.github.com/articles/about-pull-requests/)
+    with a clear title and description against the `main` branch.
+
+> By submitting your changes to Charmed HPC's documentation, you agree to license your
+> contributions under the terms of the [Creative Commons Attribution-Share Alike
+> International license, version 4.0](./LICENSE).
```

Comment:
Is this necessary? There's already a license section below.

Reply:
Eh, not really, it's just something that I've carried from the original CONTRIBUTING.md template I put together like 5 years ago :skull: 

I think it we can be DRY here as we're effectively repeating the same blurb later within the document.

```suggestion
> By submitting your changes to Charmed HPC's documentation, you agree to license your
> contributions under the terms of the [Creative Commons Attribution-Share Alike
> International license, version 4.0](./LICENSE).
```